components/JournalEntryList/index.js

```diff
@@ -0,0 +1,44 @@
+import { titles } from "/lib/data.js";
+import styled from "styled-components";
+
+export default function JournalEntryList() {
+  return (
+    <>
+      <header>
+        <Heading>Your Entries</Heading>
+      </header>
+      <main>
+        <EntryItem>
```

Comment:
The name of this component is EntryItem, but it is a list. Does the name make sense?